front/src/applications/operationalStudies/__tests__/sampleData.ts

```diff
@@ -551,3 +555,136 @@ export const trainSummaryHonored: Extract<SimulationSummaryResult, { status: 'su
   path_item_times_provisional: [0, 2186885],
   path_item_times_base: [0, 2186885],
 };
+
+// Data for upsertMapWaypointsInOperationalPoints
+
+export const pathInputsWithOneMapWaypoint: TrainScheduleResult['path'] = [
+  {
+    id: '1',
+    secondary_code: 'BV',
+    uic: 2,
+  },
+  {
+    id: '2',
+    offset: 7746000,
+    track: 'TA6',
+  },
+  {
+    id: '3',
+    secondary_code: 'BV',
+    uic: 4,
+  },
+];
+
+export const pathInputsWithNoMapWaypoint: TrainScheduleResult['path'] = [
+  {
+    id: '1',
+    secondary_code: 'BV',
+    uic: 2,
+  },
+  {
+    id: '2',
+    secondary_code: 'BV',
+    uic: 3,
+  },
+  {
+    id: '3',
+    secondary_code: 'BV',
+    uic: 4,
+  },
+];
+
+export const sampleWithMultipleOperationalPoints: NonNullable<
+  PathProperties['operational_points']
+> = [
+  {
+    id: 'West_station',
+    part: {
+      track: 'TA1',
+      position: 500,
+    },
+    extensions: {
+      identifier: {
+        name: 'West_station',
+        uic: 2,
+      },
+    },
+    position: 0,
+  },
+  {
+    id: 'Mid_West_station',
+    part: {
+      track: 'TC1',
+      position: 550,
+    },
+    extensions: {
+      identifier: {
+        name: 'Mid_West_station',
+        uic: 3,
+      },
+    },
+    position: 12050000,
+  },
+  {
+    id: 'Mid_East_station',
+    part: {
+      track: 'TD0',
+      position: 14000,
+    },
+    extensions: {
+      identifier: {
+        name: 'Mid_East_station',
+        uic: 4,
+      },
+    },
+    position: 26500000,
+  },
+];
+
+export const pathInputWithWaypointsByMapOnly: TrainScheduleResult['path'] = [
+  {
+    id: '1',
+    offset: 6481000,
+    track: 'TA6',
+  },
+  {
+    id: '2',
+    offset: 4733000,
+    track: 'TA6',
+  },
+];
+
+export const pathInputsContaingOneOperationPoint: TrainScheduleResult['path'] = [
+  {
+    id: '1',
+    offset: 6481000,
+    track: 'TA6',
+  },
+  {
+    id: '2',
+    offset: 679000,
+    track: 'TC0',
+  },
+  {
+    id: '3',
+    offset: 883000,
+    track: 'TC0',
+  },
+];
```

Comment:
There is no OP in this path :)

Reply:
Oh yes it's not very clear, I meant the path will pass through an operational point. I renamed it accordingly to what we test with this sample.